be2-scala/docs/README.md

```diff
@@ -303,7 +305,26 @@ final case class DbActorReadLaoDataAck(laoData: LaoData) extends DbActorMessage
 
 For the Social Media functionality, each user has their own channel with the identifier `root/lao_id/own_pop_token` and each broadcast containing the message_id of a post will be written to `root/lao_id/posts`.
 
+For the Election functionality, we need to have a key pair stored safely somewhere so that we can encrypt/decrypt messages. That is why we use a `ELectionData` object to store the key pairs for the corresponding election.
+The path root is `root/private/election_id` as stated above.
+The key pair can be stored and retrieved by the following functions.
 
+```scala
+final case class CreateElectionData(id: Hash, keyPair: KeyPair) extends Event
+final case class ReadElectionData(electionId: Hash) extends Event
+
+final case class DbActorReadElectionDataAck(electionData: ElectionData) extends DbActorMessage
+```
+
+The RollCallData is an object that stores the id and state of the previous rollcall action (`CREATE`, `OPEN`, `REOPEN`, `CLOSE`). It ensures that we cannot open a closed rollcall or close a non-opened rollcall.
```

Comment:
id and state of the ... `previous rollcall action` or `latest roll call` ?

Reply:
Yup `latest` is more adequate here.